src/main/kotlin/com/terraformation/backend/accelerator/AcceleratorProjectService.kt

```diff
@@ -0,0 +1,38 @@
+package com.terraformation.backend.accelerator
+
+import com.terraformation.backend.accelerator.model.AcceleratorProjectModel
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.accelerator.tables.references.COHORTS
+import com.terraformation.backend.db.accelerator.tables.references.PARTICIPANTS
+import com.terraformation.backend.db.accelerator.tables.references.PROJECT_VOTE_DECISIONS
+import com.terraformation.backend.db.default_schema.tables.references.PROJECTS
+import jakarta.inject.Named
+import org.jooq.DSLContext
+
+@Named
+class AcceleratorProjectService(
```

Comment:
We can likely replace this class with just SearchService functionalities. We will need to add Cohort and Participant search tables.

Reply:
I think this is a bit better since (a) it gives us a strongly-typed return value which means IntelliJ can do autocomplete and such when we're editing templates, and (b) the query is more compact and less costly both to generate and to run.